eclipsePlugin/build.gradle

```diff
@@ -34,6 +36,12 @@ def eclipsePluginId = 'com.github.spotbugs.plugin.eclipse'
 def localProps = new Properties()
 localProps.load(new FileInputStream("$projectDir/local.properties"))
 
+ext {
+  eclipseRootDir = localProps.getProperty('eclipseRoot.dir')
+  eclipseExecutable = eclipseRootDir + (Os.isFamily(Os.FAMILY_MAC) ? "/Contents/MacOS/eclipse" : "/eclipse")
+  eclipsePluginsDir = eclipseRootDir + (Os.isFamily(Os.FAMILY_MAC) ? "/Contents/Eclipse/plugins" : "/plugins")
+}
```

Comment:
Sounds nice, but according to #203, 4.3 and 4.4 follows the same directory layout with 3.6. Then we need to drop support not only for 3.6 & 3.8 but also 4.3 and 4.4.

- [x] Oxygen (4.7)
- [x] Neon (4.6)
- [x] Mars (4.5) which starts new directory layout
- [x] Luna (4.4)
- [x] Kepler (4.3)
- [ ] Juno (4.2, 3.8)
- [ ] Previous to Juno (<=4.1)